packages/docprovider/src/awareness.ts

```diff
@@ -0,0 +1,102 @@
+// From https://en.wikipedia.org/wiki/Moons_of_Jupiter
+export const moonsOfJupyter = [
+  'Metis',
+  'Adrastea',
+  'Amalthea',
+  'Thebe',
+  'Io',
+  'Europa',
+  'Ganymede',
+  'Callisto',
+  'Themisto',
+  'Leda',
+  'Ersa',
+  'Pandia',
+  'Himalia',
+  'Lysithea',
+  'Elara',
+  'Dia',
+  'Carpo',
+  'Valetudo',
+  'Euporie',
+  'Eupheme',
+  // 'S/2003 J 18',
+  // 'S/2010 J 2',
+  'Helike',
+  // 'S/2003 J 16',
+  // 'S/2003 J 2',
+  'Euanthe',
+  // 'S/2017 J 7',
+  'Hermippe',
+  'Praxidike',
+  'Thyone',
+  'Thelxinoe',
+  // 'S/2017 J 3',
+  'Ananke',
+  'Mneme',
+  // 'S/2016 J 1',
+  'Orthosie',
+  'Harpalyke',
+  'Iocaste',
+  // 'S/2017 J 9',
+  // 'S/2003 J 12',
+  // 'S/2003 J 4',
+  'Erinome',
+  'Aitne',
+  'Herse',
+  'Taygete',
+  // 'S/2017 J 2',
+  // 'S/2017 J 6',
+  'Eukelade',
+  'Carme',
+  // 'S/2003 J 19',
+  'Isonoe',
+  // 'S/2003 J 10',
+  'Autonoe',
+  'Philophrosyne',
+  'Cyllene',
+  'Pasithee',
+  // 'S/2010 J 1',
+  'Pasiphae',
+  'Sponde',
+  // 'S/2017 J 8',
+  'Eurydome',
+  // 'S/2017 J 5',
+  'Kalyke',
+  'Hegemone',
+  'Kale',
+  'Kallichore',
+  // 'S/2011 J 1',
+  // 'S/2017 J 1',
+  'Chaldene',
+  'Arche',
+  'Eirene',
+  'Kore',
+  // 'S/2011 J 2',
+  // 'S/2003 J 9',
+  'Megaclite',
+  'Aoede',
+  // 'S/2003 J 23',
+  'Callirrhoe',
+  'Sinope'
+];
+
+export const getRandomUserName = (): string =>
+  moonsOfJupyter[Math.floor(Math.random() * moonsOfJupyter.length)];
+
+/**
+ * Default colors of the Y Project licensed under the CC-BY Creative Commons. https://creativecommons.org/licenses/by/4.0/
+ */
+export const usercolors = [
+  { color: '#30bced', light: '#30bced33' },
+  { color: '#6eeb83', light: '#6eeb8333' },
+  { color: '#ffbc42', light: '#ffbc4233' },
+  { color: '#ecd444', light: '#ecd44433' },
+  { color: '#ee6352', light: '#ee635233' },
+  { color: '#9ac2c9', light: '#9ac2c933' },
+  { color: '#8acb88', light: '#8acb8833' },
+  { color: '#1be7ff', light: '#1be7ff33' }
+];
+
+export const getRandomColor = (): { color: string; light: string } =>
```

Comment:
Is the list of colors long enough to help pick a distinct color?

Reply:
No it's not. These are just a few colors I like to use in my projects. But they don't provide nice contrast for Jupyterlab. I really think that we need to change them to something better. But this is better than nothing.